### ISSUE TITLE
MA0016 should be disabled for implicit operators

### ISSUE THREAD

**Issue by wiz0u** [meziantou/Meziantou.Analyzer#882]
### Version of the Meziantou.Analyzer NuGet package

2.0.215

### Rule Identifier

MA0016

### Target Framework

netstandard2.0;net6.0

### C# Language version

C# 11

### Description

MA0016 suggests using collection abstraction (interface), but arguments cannot be interfaces for implicit operators.

### Reproduction Steps

Minimal code:

```c#
code to reproduce the error
```

**Comment by meziantou** [comment]
Thanks for reporting it! The fix should be available very soon on nuget.org (https://github.com/meziantou/Meziantou.Analyzer/actions/runs/18252367657)
